Timer/MVMainView.swift

```diff
@@ -10,55 +21,53 @@ class MVMainView: NSView {
     get{return self.contextMenu}
     set{}
   }
-  
+
   override init(frame frameRect: NSRect) {
     super.init(frame: frameRect)
-    
+
     self.contextMenu = NSMenu(title: "Menu")
     menuItem = NSMenuItem(title:"Show in Dock", action:#selector(self.toggleShowInDock), keyEquivalent:"")
     self.contextMenu?.addItem(menuItem!)
-    
+
     let nc = NotificationCenter.default
     nc.addObserver(self, selector: #selector(windowFocusChanged), name: NSWindow.didBecomeKeyNotification, object: nil)
     nc.addObserver(self, selector: #selector(windowFocusChanged), name: NSWindow.didResignKeyNotification, object: nil)
   }
-  
+
   required init?(coder: NSCoder) {
     fatalError("init(coder:) has not been implemented")
   }
-  
+
   @objc func toggleShowInDock() {
     if menuItem?.state == .on {
       appDelegate.removeBadgeFromDock()
     } else {
       appDelegate.addBadgeToDock(controller: self.controller!)
     }
   }
-  
+
   deinit {
     let nc = NotificationCenter.default
     nc.removeObserver(self)
   }
-  
+
   override func draw(_ dirtyRect: NSRect) {
     super.draw(dirtyRect)
     
-    let windowHasFocus = self.window?.isKeyWindow ?? false
-    var topColor = NSColor(srgbRed: 0.949, green: 0.9451, blue: 0.949, alpha: 1.0)
-    var bottomColor = NSColor(srgbRed: 0.8392, green: 0.8314, blue: 0.8392, alpha: 1.0)
```

Comment:
Did you intentionally remove the gradient?
I prefer having a gradient, since it makes the app feel more like a physical object. You could track both the "background-color-top" and "background-color-bottom" in the asset catalog.